### PR TITLE
#162099431 fix access denial to the database by PostgreSQL

### PR DESCRIPTION
**Pull request by matthenge**
#### What does this PR do?
fix the denial of access to the database due to postgres user authentication failure.
#### Description of Task to be completed?
.Override the default postgres role and set password.
.Add password to the URL
#### How should this be manually tested?
.Clone the repository to your local machine
.Cd into the directory 
.Test using Postman
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/162099431

### CODE REVIEW

**Comment by cosmas28** [comment]
Kindly fix Travis CI issues before merging this PR.

File: app/db_config.py [hunk 1]
```diff
@@ -4,15 +4,19 @@
...
 db_url = os.getenv('DB_URL')
-conn = psycopg2.connect('db_url')
+def connection(db_url):
+    conn = psycopg2.connect(db_url)
+    return conn
+
+
 
 
 def init_db():
```
Comment (cosmas28) [inline app/db_config.py:11]: Too many spaces before this function.

File: app/__init__.py [hunk 2]
```diff
@@ -1,8 +1,10 @@
 from flask import Flask, Blueprint
 from .api.v2 import version2 as v2
+from .db_config import create_tables
 
 def create_app():
     app = Flask(__name__)
     app.register_blueprint(v2)
+    create_tables()
 
     return app
```
Comment (cosmas28) [inline app/__init__.py]: kindly leave only one line space at end of each file.

**Comment by Kitingu** [comment]
Try to fix Travis build before merging.

**Pull request merged**